apps/src/music/player/MusicLibrary.ts

```diff
@@ -69,6 +67,18 @@ export default class MusicLibrary {
   }
 }
 
+export type LibraryJson = {
+  groups: FolderGroup[];
+};
+
+export const LibraryValidator: ResponseValidator<LibraryJson> = response => {
+  const libraryJson = response as LibraryJson;
+  if (!libraryJson.groups || libraryJson.groups.length === 0) {
+    throw new Error(`Invalid library JSON: ${response}`);
+  }
+  return libraryJson;
+};
```

Comment:
Is this used?

Reply:
Oops forgot to actually pass this - good catch!